daemon/graphdriver/plugin.go

```diff
@@ -12,6 +13,9 @@ import (
 )
 
 func lookupPlugin(name string, pg plugingetter.PluginGetter, config Options) (Driver, error) {
+	if os.Getenv("DOCKERD_DEPRECATED_GRAPHDRIVER_PLUGINS") == "" {
+		return nil, fmt.Errorf("DEPRECATED: Experimental graphdriver plugins are deprecated, and disabled by default. This feature will be removed in the next release. See https://docs.docker.com/go/deprecated/")
```

Comment:
nit: 🙈 But looks like linter doesn't complain, so let's just leave it as is
```suggestion
		return nil, errors.New("DEPRECATED: Experimental graphdriver plugins are deprecated, and disabled by default. This feature will be removed in the next release. See https://docs.docker.com/go/deprecated/")
```

Reply:
OH! That one will for sure pop-up once I updated golangci-lint. Current version of it will barf on those. But I expect there's gonna be many to fix, and I didn't spend time yet on updating it (might be a fun weekend project 😂)